docs/conf.py

```diff
@@ -270,7 +265,7 @@
 napoleon_google_docstring = True
 napoleon_numpy_docstring = False  # Explicitly prefer Google style docstring
 napoleon_use_param = True  # for type hint support
-
+napoleon_use_rtype = False  # False so the return type is inline with the description.
```

Comment:
So for reference this looks like

https://jira--1474.org.readthedocs.build/api.html#jira.client.JIRA.add_group


Before:
![image](https://user-images.githubusercontent.com/26027314/186926720-919fc79c-f784-4ec0-b9fa-f345d50798e9.png)

After:
![image](https://user-images.githubusercontent.com/26027314/186926914-2a41c656-b15e-45e1-b550-7e65e54743a6.png)